contrib/Dockerfile.manylinux

```
@@ -149,8 +149,8 @@ RUN cd /workspace && \
 
 # The base image libcurl is linked against openssl 1.x, so we need to build from source
 # in order to use openssl 3.x. This is needed to build aws-sdk-cpp.
-RUN wget https://curl.se/download/curl-8.5.0.tar.gz && \
-    tar xzf curl-8.5.0.tar.gz && cd curl-8.5.0 && \
+RUN wget https://curl.se/download/curl-8.19.0.tar.gz && \
```

Hadolint (2.14.0)

[info] Avoid use of wget without progress bar. Use `wget --progress=dot:giga <url>`. Or consider using `-q` or `-nv` (shorthands for `--quiet` or `--no-verbose`).

(DL3047)